automathor.py

```
@@ -0,0 +1,170 @@
+
+
+@dataclass
+class Context:
+    filename: str
+    line: int
+    commit: str
+    author_email: str
+    datetime: datetime
+    todo: str
+    text: str
+    lines: List[AnnotatedLine]
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)